lib/node_modules/@stdlib/stats/base/dists/t/kurtosis/README.md

```diff
@@ -135,6 +135,98 @@ for ( i = 0; i < 10; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dists/t/kurtosis.h"
+```
+
+#### stdlib_base_dists_t_kurtosis( v )
+
+Returns the excess kurtosis of a Student's t distribution.
+
+```c
+double out = stdlib_base_dists_t_kurtosis( 9.0 );
+// returns 1.2
+```
+
+The function accepts the following arguments:
+
+-   **v**: `[in] double` degrees of freedom.
+
+```c
+double stdlib_base_dists_t_kurtosis( const double v );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/stats/base/dists/t/kurtosis.h"
+#include <stdlib.h>
+#include <stdio.h>
+
+static double random_uniform( const double min, const double max ) {
+    double v = (double)rand() / ( (double)RAND_MAX + 1.0 );
+    return min + ( v*(max-min) );
+}
+
+int main( void ) {
+    double v;
+    double y;
+    int i;
+
+    for ( i = 0; i < 25; i++ ) {
+        v = random_uniform( 0, 20 );
```

Comment:
@aayush0325 As `random_uniform` takes `doubles`, our convention is to always add decimals, even for integer values. Otherwise, `0` is an int which then must be implicitly cast to a double. We try to always make types explicit. This applies here and in other PRs.

```suggestion
        v = random_uniform( 0.0, 20.0 );
```

Reply:
Also applies in C benchmarks and `example.c`.